.vscode/launch.json

```diff
@@ -0,0 +1,25 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+
+
+        {
+            "name": "Run too with local config",
+            "type": "go",
+            "request": "launch",
+            "mode": "debug",
+            "program": "${workspaceRoot}",
+            "args": [
+                "apigateway", "enumerate", "--region", "us-east-1", "-o", "json"
```

Comment:
Probally dont want to check this in

Reply:
i think it is fine just to give an example.